remote_control.py

```diff
@@ -110,3 +120,23 @@ def constrain(self, val, min_val, max_val):
     def stop(self):
         logging.info("RC challenge stopping")
         self.killed = True
+
+    def dither(self, counter, speed):
+        if self.deadband < abs(speed) < self.slow_speed:
+            speed = int(speed - math.copysign(1, speed))
+            counter += 1
+            if counter < self.boost_cycles:
+                speed = speed + int(math.copysign(self.slow_speed, speed))
+            if counter > (self.boost_cycles + self.boost_dwell):
+                counter = 0
+        return (counter, speed)
+
+    def exp(self, demand, exp):

```

Comment:
This method might be useful elsewhere. how would we achieve that?

Reply:
as mentioned for dither, I think this will stay with wherever the joystick input is captured. it could be put in its own library/module but I don't think that's necessary. I thought its current form was relatively self contained if it was wanted to be duplicated/moved to another library.